### ISSUE TITLE
ELK version 6.2.3

### ISSUE THREAD

**Issue by mliljedahl** [wazuh/wazuh-kibana-app#341]
Not sure how tight Wazuh is tied to patch versions like the resent update of the ELK stack from 6.2.2 to 6.2.3. And why the Wazuh plugin stoped working because of the patch;

```kibana: {"type":"log","@timestamp":"2018-03-26T07:31:36Z","tags":["plugin","warning"],"pid":6347,"message":"Plugin \"wazuh\" was disabled because it expected Kibana version \"6.2.2\", and found \"6.2.3\"."}```

**Comment by JuanjiJG** [comment]
Hello @mliljedahl,

The team is working on adapting the Wazuh app to the latest version of the Elastic Stack, v6.2.3. In the meantime, you can use our latest available Wazuh app for Elastic v6.2.2.

Follow these steps **as superuser**:

1. Stop Kibana:
```bash
systemctl stop kibana
```

2. Remove the old Wazuh app plugin:
```bash
/usr/share/kibana/bin/kibana-plugin remove wazuh
```

3. Remove the optimizing bundles:
```bash
rm -rf /usr/share/kibana/optimize/bundles
```

4. Increase the default Node.js memory heap to avoid installation problems:
```bash
export NODE_OPTIONS="--max-old-space-size=3072"
```

5. Install the new Wazuh app package:
```bash
/usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/wazuhapp/wazuhapp-3.2.1_6.2.2.zip
```
Wait until the app installation process finishes.

6. Restart Kibana:
```bash
systemctl restart kibana
```

Now you should have the app properly running and appearing on the left sidebar on the Kibana interface.

Let me know if everything gets solved with these steps, or if you're still having problems.

Regards,
Juanjo

**Comment by mliljedahl** [comment]
Thanks!

Works great except for that the plugin url in step 5 should be: `/usr/share/kibana/bin/kibana-plugin install https://packages.wazuh.com/wazuhapp/wazuhapp-3.2.1_6.2.3.zip`

//Markus

**Comment by JuanjiJG** [comment]
Hello again @mliljedahl.

Apologies, I think I misunderstood your problem. Looks like you already had Kibana 6.2.3, and you were trying to install our latest app, available for v6.2.2.

We've finished the final tests and we just released a new version a couple of hours ago, with full support for Kibana 6.2.3. I assume that when you proceeded to install the package, it was already available to the public. 😄 

Feel free to open a new issue if you have problems or questions with the Wazuh app.

Best,
Juanjo
